frontend/src/api/k8s/notebooks.ts

```diff
@@ -29,6 +35,97 @@ import { getImageStreamDisplayName } from '~/pages/projects/screens/spawner/spaw
 import { k8sMergePatchResource } from '~/api/k8sUtils';
 import { getshmVolume, getshmVolumeMount } from '~/api/k8s/utils';
 
+export const assembleNotebookImage = (
```

Comment:
This is very similer to the assembleNotebook function. what is the reason for this one